### ISSUE TITLE
Trailing stop for crypto/usdt pairings

### ISSUE THREAD

**Issue by melvintjc96** [Speedy765/crypto-helper-tools#16]
Hi, i would like to know if it is possible to implement trailing stop loss function for (insert coin here)/usdt fiat pairing on top of the mentioned (insert coin here)/btc pairings? Thanks for your attention.

**Comment by Speedy765** [comment]
Yes that is possible. It requires some changes because your balance is usdt instead of btc in that case.

Sorry for the late reply, was on holiday.
